index.rst

```diff
@@ -4,9 +4,173 @@ Collimated Beam Projector Installation and ComCam Testing
 
 .. abstract::
 
-   Report on the initial installation and testing of the laser and Collimated Beam Projector.
+   Report on the initial installation and testing of the laser and Collimated Beam Projector during the ComCam campaign.
 
-Add content here
-================
+Introduction
+============
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+The Collimated Beam Projector (CBP) is a Schmidt Reflector run in reverse. We use an Ekspla tunable laser connected to an integrating sphere to illuminate the CBP focal plane. We design custom masks that we place at the focal plane and then CBP sends out collimated beams of light that can be reimaged by ComCam. The beamsize is 240 mm, so we illuminate only part of the Simonyi Telescope mirror. However, by doing a copointing dance, we can illuminate any part of the mirror and any point on the focal plane. A NIST calibrated photodiode in the integrating sphere allows us to know exactly how much light we are sending into the telescope, allowing us to perform very precise photometry of our spots.
+
+The main goal of the CBP is to measure the throughput of collimated light with high precision, but more generally with the CBP we can:
+
+- Measure the wavelength-dependent throughput of collimated light.
+- Look for reflectance and transmission variations across the mirrors and filters.
+- Study ghosting.
+- Measure cross-talk nonlinearity.
+- Look at pointing offsets created on parts of the mirrors when AOS applies bending mode corrections.
+
+CBP Design
+==========
+
+The CBP, a Schmidt reflector, has a focal length of 625 mm and an aperture of 240 mm. It has slots for five different masks. We use an Ekspla NT242 tunable laser fiber fed into a 6" integrating sphere to illumminate the focal plane. A NIST-calibrated photodiode in the integrating sphere is read out by an electrometer and can then be used to determine the amount of flux sent to the Simonyi Telescope (see `SITCOMTN-106 <https://sitcomtn-106.lsst.io/#time-synchronization-for-the-cbp-calibration-system>`_).
+
+.. container:: figure
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 0
+
+      * - .. image:: images/CBP_preshipment.png
+            :width: 300px
+        - .. image:: images/mask_holders.png
+            :width: 300px
+
+   **Figure 1**: The CBP in the lab in Tuscon before shipment (left) and the CBP mask holder (right).
+ 
+CBP Repair
+==========
+
+The CBP arrived on the summit badly damaged. The dew shield that held the Schmidt corrector fell off during transit, many screws came loose, the mask changer stage drive arm broke, the azimuth drive shaft was damaged, the azimuth and elevation disks were damaged, and there were metal shavings and sawdust everywhere. 
```

Comment:
During the shipment of the CBP from Tucson to Rubin Observatory, it suffered from considerable damage. When the crate was opened, it was found that the dew shield that held the Schmidt correct fell off and there were metal shavings and sawdust everywhere. The CBP was transferred down to La Serena where a more detailed evaluation took place. Damage was found on the mask changer stage drive arm, azimuth drive shaft, and both azimuth and elevation drive disks.